### ISSUE TITLE
pt-PT translation effort

### ISSUE THREAD

**Issue by gvlx** [todogroup/todogroup.org#317]
Following the work done on #197 by @puppe1990, I'm starting the Portuguese/Portugal translation.

Tasks:
- [ ] fork repo
- [ ] create pt-PT branch
- [ ] create pt-PT directory structure
- [ ] first level translation of about.md file using GT or Deepl and the pt-BR source.
- [ ] create a Slack channel "translation pt-PT"
- [ ] create a community Working Group to work on this (esop.pt support?)

Note to self: find a translation tool to help with the effort (Open Source if possible). --> this could help with resyncing of the pt-BR branch

**Comment by anajsana** [comment]
hey, @gvlx if you are not able to connect with @puppe1990 (I haven't been able either), feel free to kick off this, and I'll review the PR once this is finished 👍

**Comment by anajsana** [comment]
Also, we are adding the translations to the evolution of the OSPO study in the "guide and resources" section. In case you'd like to make a Portuguese contribution 😄 @gvlx 

![Screenshot 2022-05-25 at 12-11-19 TODO Group __ TODO Talk openly develop openly](https://user-images.githubusercontent.com/43671777/170238809-15e35d87-680c-4ccb-a6db-e2a992970bed.png)

**Comment by anajsana** [comment]
PS: We are working to redesign the entire website to improve accessibility, so this issue will become outdated soon. @gvlx do you want to wait until the working branch is merged to the main branch and continue with the translation efforts later or should this issue be closed directly and marked as won't fix?
